samples/excel/85-preview-apis/data-types-formatted-number.yaml

```diff
@@ -0,0 +1,149 @@
+order: 1
+id: excel-data-types-formatted-number
+name: 'Data types: Formatted numbers'
+description: This sample shows how to set and get data types using the formatted number properties.
+host: EXCEL
+api_set:
+    ExcelApi: BETA (PREVIEW ONLY)
+script:
+    content: |
+        $("#set-formatted-number-date").click(() => tryCatch(setFormattedNumberDate)); 
+        $("#set-formatted-number-currency").click(() => tryCatch(setFormattedNumberCurrency)); 
+        $("#get-formatted-number").click(() => tryCatch(getFormattedNumber));
+
+        async function setFormattedNumberDate() {
+          await Excel.run(async (context) => {
+            // This method creates a formatted number data type,
+            // and sets the format of this data type as a date.
+
+            // Create a worksheet called "Sample" and activate it.
+            context.workbook.worksheets.getItemOrNullObject("Sample").delete();
+            const sheet = context.workbook.worksheets.add("Sample");
+            sheet.activate();
+            
+            // Get cell A1 and write a number formatted as a date to cell A1.
+            const dateRange = sheet.getRange("A1");
+            dateRange.valuesAsJson = [
+              [
+                {
+                  type: Excel.CellValueType.formattedNumber,
+                  basicValue: 32889.0,
+                  numberFormat: "m/d/yyyy"
```

Comment:
General feature feedback: This does not override the standard `Range.numberFormats`. If you click one of the formatting options in the Excel UI, it'll change the cell without the underlying JSON numberFormat changing. I found this confusing.
![image](https://user-images.githubusercontent.com/38896772/149412655-fb557dc6-6031-4f71-a4f4-c6ee52fb6079.png)

Reply:
This is a great point. I will take this feedback to the feature crew.